multicast/src/test/kotlin/com/dropbox/flow/multicast/ChannelManagerTest.kt

```diff
@@ -141,6 +140,40 @@ class ChannelManagerTest {
             assertThat(collection1.await()).isEqualTo(listOf("a", "b"))
             assertThat(collection2.await()).isEqualTo(listOf("a", "b"))
         }
+
+    @Test
+    fun `Calling close on ChannelManager should close downstream channels when a value is also dispatched`() =
```

Comment:
nit: formatting the name in GIVEN x WHEN y THEN z will make it easier to transform these tests to a more sane BDD format one day when we're not trying to avoid taking in junit5 deps. in the future we would be able to write tests like:
```
class MyTests({
  GIVEN("a state") {
    // set up state
    WHEN("something happens") {
      // make something happened
      THEN("expect a result")
        // verify result
    }
    WHEN("something else happens") {
      // make something else happened
      THEN("expect another result")
        // verify another result
    }
  }
})